app/views/layouts/_header.html.erb

```diff
@@ -84,7 +84,7 @@
         <% else %>
           <li>
             <% ab_test(:signup_button_size, 'btn btn-large', '') do |button_size| %>
-            <%= link_to 'Join', new_veteran_registration_path, class: button_size, style: 'background-color: #B23900;' %>
+            <%= link_to 'Join', new_veteran_registration_path, class: 'btn', style: 'background-color: #26A69A;' %>
```

Comment:
What about this comment? Should we remove the test altogether?
https://github.com/OperationCode/operationcode/issues/637#issuecomment-281114407